### DISCUSSION TITLE
Hindi fonts are not displaying on a canvas when using Puppeteer with headless Chromium on a Linux OS

### DISCUSSION THREAD

**Discussion by RathodAnnu13** [fabricjs/fabric.js#9358] [Q&A]
I am using Puppeteer to open headless browser and run fabric canvas on it.
While passing text: 'शहर' in i-text . it display on windows as expected but not in linux server.

**Comment by ShaMan123** [comment]
It is not related to us, you need to register fonts in node canvas
https://github.com/Automattic/node-canvas#registerfont

BTW We have started working with playwright... it seems great.

**Reply by asturur** [reply]
If he is using puppeteer probably node-canvas is not being used, but still @RathodAnnu13 you have to be sure the font is available on that linux server. Said so i do not know how to add fonts to a linux server outside ubuntu UI
